### PR TITLE
i18n(es): Update `tutorial` & fix typos

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Update `tutorial` [`0b2ebea`](https://github.com/withastro/docs/commit/0b2ebea9fa080c30bd1b0308326fbc025b4e9073#diff-d4b2821dbf3833fa9a439e77e9af6b66f6286435054cee44ffdd2b027430e9ab)

### CODE REVIEW

**Comment by Waxer59** [comment]
add commit [`f607140`](https://github.com/withastro/docs/commit/f607140fb45e30a3adb212b85100d7bcb419a345)

File: src/content/docs/es/tutorial/2-pages/2.mdx [hunk 1]
```diff
@@ -61,26 +61,26 @@ Ahora que ya has creado páginas con archivos `.astro`, vamos a crear algunos po
...
 
 3. Utiliza las herramientas de desarrollo de tu navegador para inspeccionar esta página. Observa que, aunque no has escrito ningún elemento HTML, tu Markdown se ha convertido a HTML. Puedes ver elementos como encabezados, párrafos y elementos de lista.
 
 :::note
-La información en la parte superior del archivo, dentro de las vallas de código, se llama frontmatter. Estos datos -incluyendo etiquetas y una imagen del post- son información *sobre* tu post que Astro puede usar. No aparece en la página automáticamente, pero accederemos a ella más adelante en el tutorial para mejorar tu sitio. 
+La información en la parte superior del archivo, dentro de las vallas de código, se llama frontmatter. Estos datos -incluyendo etiquetas y una imagen de la publicación- son información *sobre* tu publicación que Astro puede usar. No aparece en la página automáticamente, pero accederás a ella más adelante en el tutorial para mejorar tu sitio.
```
Comment (dreyfus92) [inline src/content/docs/es/tutorial/2-pages/2.mdx:78]:
```suggestion
La información en la parte superior del archivo, dentro de las vallas de código, se llama frontmatter. Estos datos -incluyendo etiquetas y una imagen de la publicación - son información *sobre* tu publicación que Astro puede usar. No aparece en la página automáticamente, pero accederás a ella más adelante en el tutorial para mejorar tu sitio. 
```

File: src/content/docs/es/tutorial/6-islands/index.mdx [hunk 2]
```diff
@@ -14,21 +14,21 @@ import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
...
-## ¿Dónde estamos ahora?
+## ¿Dónde estás ahora?
 
 Puedes encontrar el código en este punto del tutorial en [GitHub](https://github.com/withastro/blog-tutorial-demo/tree/unit-6/start) o en
 [StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/unit-6/start?file=src/pages/index.astro).
 
 Revisa tu código con este ejemplo, o si acabas de unirte a nosotros, haz un fork en StackBlitz para empezar a codificar en tu navegador desde aquí.
 
-## ¿Adónde vamos?
+## ¿Adónde vas?
```
Comment (dreyfus92) [inline src/content/docs/es/tutorial/6-islands/index.mdx:26]:
```suggestion
## ¿A dónde vas?
```

File: src/content/docs/es/tutorial/2-pages/index.mdx [hunk 3]
```diff
@@ -4,30 +4,30 @@ unitTitle: 'Añadir, estilizar y enlazar páginas en su sitio web'
...
-## ¿Dónde estamos ahora?
+## ¿Dónde estás ahora?
 
 Puedes encontrar el código en este punto del tutorial en [GitHub](https://github.com/withastro/blog-tutorial-demo/tree/unit-2/start) o en
 [StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/unit-2/start?file=src/pages/index.astro).
 
 Revisa tu código con este ejemplo, o si acabas de unirte a nosotros, haz un fork en StackBlitz para empezar a codificar en tu navegador desde aquí.
 
-## ¿Adónde vamos?
+## ¿Adónde vas?
```
Comment (dreyfus92) [inline src/content/docs/es/tutorial/2-pages/index.mdx:24]:
```suggestion
## ¿A dónde vas?
```

File: src/content/docs/es/tutorial/3-components/index.mdx [hunk 4]
```diff
@@ -14,16 +14,16 @@ import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
...
-## ¿Dónde estamos ahora?
+## ¿Dónde estás ahora?
 
 Puedes encontrar el código en este punto del tutorial en [GitHub](https://github.com/withastro/blog-tutorial-demo/tree/unit-3/start) o en
 [StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/unit-3/start?file=src/pages/index.astro).
 
 Revisa tu código con este ejemplo, o si acabas de unirte a nosotros, haz un fork en StackBlitz para empezar a codificar en tu navegador desde aquí.
 
-## ¿Adónde vamos?
+## ¿Adónde vas?
```
Comment (dreyfus92) [inline src/content/docs/es/tutorial/3-components/index.mdx:26]:
```suggestion
## ¿A dónde vas?
```

File: src/content/docs/es/tutorial/4-layouts/index.mdx [hunk 5]
```diff
@@ -4,35 +4,35 @@ unitTitle: Ahorra tiempo y energía con plantiallas de página reutilizables
...
-## ¿Dónde estamos ahora?
+## ¿Dónde estás ahora?
 
 Puedes encontrar el código en este punto del tutorial en [GitHub](https://github.com/withastro/blog-tutorial-demo/tree/unit-4/start) o en
 [StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/unit-4/start?file=src/pages/index.astro).
 
 Revisa tu código con este ejemplo, o si acabas de unirte a nosotros, haz un fork en StackBlitz para empezar a codificar en tu navegador desde aquí.
 
-## ¿Adónde vamos?
+## ¿Adónde vas?
```
Comment (dreyfus92) [inline src/content/docs/es/tutorial/4-layouts/index.mdx:26]:
```suggestion
## ¿A dónde vas?
```

File: src/content/docs/es/tutorial/5-astro-api/index.mdx [hunk 6]
```diff
@@ -17,16 +17,16 @@ import Option from '~/components/tutorial/Option.astro';
...
-## ¿Dónde estamos ahora?
+## ¿Dónde estás ahora?
 
 Puedes encontrar el código en este punto del tutorial en [GitHub](https://github.com/withastro/blog-tutorial-demo/tree/unit-5/start) o en
 [StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/unit-5/start?file=src/pages/index.astro).
 
 Revisa tu código con este ejemplo, o si acabas de unirte a nosotros, haz un fork en StackBlitz para empezar a codificar en tu navegador desde aquí.
 
-## ¿Adónde vamos?
+## ¿Adónde vas?
```
Comment (dreyfus92) [inline src/content/docs/es/tutorial/5-astro-api/index.mdx:29]:
```suggestion
## ¿A dónde vas?
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo algunas sugerencias 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by DavidEGiraldo — APPROVED** [review]
LGTM 👍

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thank you everyone 🙌

**Pull request merged**